pkg/cmd/server/server.go

```diff
@@ -170,6 +191,37 @@ func (s *server) run() error {
 	return nil
 }
 
+// Assign vSphere VC credentials and configuration parameters
+func getVCConfigParams(config serverConfig, params map[string]interface{}) error {
+
+	if config.vCenter == "" {
+		return errors.New("getVCConfigParams: parameter vcenter-address not provided")
+	}
+	params[ivd.HostVcParamKey] = config.vCenter
+
+	if config.user == "" {
+		return errors.New("getVCConfigParams: parameter vcenter-user not provided")
+	}
+	params[ivd.UserVcParamKey] = config.user
+
+	passwd := os.Getenv("VC_PASSWORD")
+	if passwd == "" {
```

Comment:
Is empty password an error case? Could you verify if an empty string can be a password.

Reply:
Removed the error message and added a log instead. Since VC_PASSWORD is an environment variable instead of a server parameter, it does not show up in the usage. So wanted to log a message to indicate the env param needs to be set.